assets/stylesheets/_typography.scss

```diff
@@ -20,8 +20,8 @@ strong {
 
 .new-window {
     padding-right: 17px;
-    background: file-url("passports/new-window-link.png") right center no-repeat;
+    background: file-url("new-window-link.png") right center no-repeat;
     &:hover {
-        background-image: file-url("passports/new-window-link-blue.png");
+        background-image: file-url("new-window-link-blue.png");
```

Comment:
The `file-url` function uses the `$path` variable that's set before these sass modules are included.